Source/Core/DolphinQt/Settings/GameCubePane.cpp

```diff
@@ -53,59 +45,69 @@ GameCubePane::GameCubePane()
 
 void GameCubePane::CreateWidgets()
 {
-  QVBoxLayout* layout = new QVBoxLayout;
+  QVBoxLayout* layout = new QVBoxLayout(this);
 
   // IPL Settings
-  QGroupBox* ipl_box = new QGroupBox(tr("IPL Settings"));
-  QGridLayout* ipl_layout = new QGridLayout;
+  QGroupBox* ipl_box = new QGroupBox(tr("IPL Settings"), this);
+  QGridLayout* ipl_layout = new QGridLayout(ipl_box);
   ipl_box->setLayout(ipl_layout);
 
-  m_skip_main_menu = new QCheckBox(tr("Skip Main Menu"));
-  m_override_language_ntsc = new QCheckBox(tr("Override Language on NTSC Games"));
-  m_language_combo = new QComboBox;
+  m_skip_main_menu = new QCheckBox(tr("Skip Main Menu"), ipl_box);
+  m_override_language_ntsc = new QCheckBox(tr("Override Language on NTSC Games"), ipl_box);
+  m_language_combo = new QComboBox(ipl_box);
+  m_language_combo->setCurrentIndex(-1);
 
   // Add languages
-  for (const auto& language :
-       {tr("English"), tr("German"), tr("French"), tr("Spanish"), tr("Italian"), tr("Dutch")})
-    m_language_combo->addItem(language);
+  for (const auto& entry : {std::make_tuple(tr("English"), 0), std::make_tuple(tr("German"), 1),
```

Comment:
Wouldn't `std::make_pair` be more appropriate here?

Should also have `#include <utility>` to not rely on headers that happen to be dragged in from others.